roadmaps/2022.md

```diff
@@ -91,33 +91,73 @@ The core team has two main goals for this quarter: to improve the product and al
+  - [ ] Fixing user pain points like pagination -> is delayed to v0.30.0, so Q4 2022.
+  - [ ] Adding really demanded features like multi-index search -> not a priority for 2022 anymore regarding the features we need to prioritize first for v1
+- [X] Start v0.30.0 implementation, including search pagination and remove user frustrations to be more production ready
 
 ### Team
 
-- [ ] Finish the recruitment process to hire a senior software engineer.
-- [ ] Onboard the new software engineer.
-- [ ] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team.
+- [X] Finish the recruitment process to hire a senior software engineer.
+- [ ] Onboard the new software engineer -> The person arrives next quarter.
+- [X] Adapt the team synchronization and rhythm according to the Product Squad (PS) freshly introduced by the product team -> We had some delay but we found out the new way of working with Product Squad better fits the needs of the team.
 
 ### Misc
 
-- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side.
-- [ ] Work with the devrel team to promote the contributions we need on the tokenizer side.
-- [ ] Keep improving indexation speed/crash
-  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement.
-  - [ ] by keep investigating and trying new technical solutions on our side.
+- [ ] Finish internship work on Jayson. Integrate it to one or two routes on the Meilisearch side. -> We don't know yet if we will prioritize it in Q4 2022 due to the huge breaking it can lead.
+- [X] Work with the devrel team to promote the contributions we need on the tokenizer side -> done during Hacktoberfest, so Q4 2022.
+- [X] Keep improving indexation speed/crash
+  - [ ] by gathering user feedback and use-cases and creating benchmarks according to them: we'll ensure we succeeded or not the improvement -> We are working on creating use-case benchmarks in Q4 2022.
+  - [X] by keep investigating and trying new technical solutions on our side -> we made a lot of changes described in [this issue](https://github.com/meilisearch/meilisearch/issues/2132).
+
+## Q4 2022
+
+This quarter is the last one we have to prepare v1 in good condition. This way, we have prioritized tasks that would help the team deliver Meilisearch v1: a stable and impactful version of Meilisearch.
+It also means we will de-prioritize implementations that could impact in a bad way the critical features of Meilisearch, like improving the indexing speed in a way that would risk making the indexing fail.
+
+Q4 is also the quarter where the Hacktoberfest is happening and this will lead us to spend time on it. We created a lot if issues during Q3, especially regarding the language support, and we hope attract the community and make them contribute in October.
+
+### v1 work
+
+- [ ] Be production ready for v1
```

Comment:
Like the focus, having it at first is nice!

Reply:
You don't mention the issue of the limit we have to put some point on the search query (Lenght/Rate Limiting/Short Circuit). This would be crucial for being "stable".

Reply:
We don't mention it specifically but it's part of 

> Ensure we don't have any bugs/crashes impacting the stability of Meilisearch. The RAM consumption should be appropriate when indexing and searching, or at least we must be transparent with the users about it."

I don't want to only focus on it, but on every similar issue we could have. We discussed during the retrospective not only this bug could happen and we want to be aware of all of them.